src/test/java/net/minecraftforge/debug/entity/player/LevelChangeEventOnEnchantTest.java

```diff
@@ -0,0 +1,21 @@
+package net.minecraftforge.debug.entity.player;
+
+import net.minecraftforge.event.entity.player.PlayerXpEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Mod("level_change_event_on_enchant_test")
+@Mod.EventBusSubscriber()
+public class LevelChangeEventOnEnchantTest {
+   private static final boolean ENABLE = true;
+   private static final Logger logger = LogManager.getLogger(LevelChangeEventOnEnchantTest.class);
+
+   @SubscribeEvent
+   public static void onPlayerLevelChange(PlayerXpEvent.LevelChange event)
+   {
+      if (!ENABLE) return;
```

Comment:
What is the point of this? If you want a flag like that for a one line body just use `if (ENABLED)`. Inverting it just to return seems silly.

Reply:
copy pasted from another xp test mod.

Reply:
Why not just add the event handler to that test mod? As they are both PlayerXPEvent...

Reply:
ah yeah could have tested using the other mod instead huh... /oops